spec/bolt/cli_spec.rb

```diff
@@ -380,14 +380,6 @@ def stub_config(file_content = {})
         cli = Bolt::CLI.new(%w[command run uptime --password opensesame --nodes foo])
         expect(cli.parse).to include(password: 'opensesame')
       end
-
-      it "prompts the user for password if not specified" do
```

Comment:
Maybe we could update these to use $future and then check on stderr? that way when we deprecate stdout we can not have to delete tests.

Reply:
These tests are for the optional parameters in the CLI, so there's no checking on stderr. Since those changes aren't gated by future I'm not sure there's a need to do this?

Reply:
Oh, my bad :man_facepalming: I thought they were for the prompt plugin. Maybe we can do that there?